Targets/CatanStats/Sources/Scenes/NewGame/Views/Subviews/PlayerNameTableViewCell.swift

```diff
@@ -0,0 +1,62 @@
+//
+//  PlayerNameTableViewCell.swift
+//  CatanStats
+//
+//  Created by Aleksandr Mamlygo on 18.06.24.
+//
+
+import UIKit
+
+final class PlayerNameTableViewCell: UITableViewCell {
+	static let reuseIdentifier = "PlayerNameCell"
+
+	private lazy var textField: UITextField = {
+		let textField = UITextField()
+		textField.borderStyle = .roundedRect
+		return textField
+	}()
+
+	override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
+		super.init(style: style, reuseIdentifier: reuseIdentifier)
+		setupUI()
+	}
+
+	required init?(coder: NSCoder) {
+		fatalError("init(coder:) has not been implemented")
+	}
+
+	override func prepareForReuse() {
+		textField.backgroundColor = .systemBackground
```

Comment:
В доках вроде написано контент обнулять в cellForRowAt(), добавить туда?
И не очень понятно относится ли цвет к контенту, кажется что нет

https://developer.apple.com/documentation/uikit/uitableviewcell/1623223-prepareforreuse
To avoid potential performance issues, you should only reset attributes of the cell that are not related to content, for example, alpha, editing, and selection state.